src/browser/documentation/help/drop-database.jsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2002-2019 "Neo4j,"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react'
+import ManualLink from 'browser-components/ManualLink'
+import DbsOnSystemDb from './partials/dbs-on-systemdb'
+const title = 'DROP DATABASE'
+const subtitle = 'Delete a database'
+const category = 'administration'
+const content = (
+  <>
+    <p>
+      Databases can be deleted using the command <code>DROP DATABASE.</code>
```

Comment:
you may change the wording around but if not just be mindful of the full stop in `<code>DROP DATABASE.</code>`

Reply:
Thank you for pointing it out! Fixed.